yarn.lock

```diff
@@ -13046,6 +13046,13 @@ postcss-logical@^3.0.0:
   dependencies:
     postcss "^7.0.2"
 
+postcss-logical@^4.0.2:
```

Comment:
Is this dependency relevant to this PR? :thinking:

Reply:
This is a generally required improvement, however, untied to this PR